cognite/client/_api_client.py

```diff
@@ -684,6 +691,12 @@ def _delete_multiple(
             task_unwrap_fn=lambda task: task["json"]["items"],
             task_list_element_unwrap_fn=utils._auxiliary.unwrap_identifer,
         )
+        if not return_items:
+            return None
+        if list_cls is None:
+            raise ValueError("If you are going to return results, then you need to pass the list_cls")
```

Comment:
This is not directed at the SDK user but at the SDK developer. I vote to make this into an `assert` instead.

Reply:
Good point, done.